### PR TITLE
[documentation]: enhance docs

### PR DESCRIPTION
**Pull request by 0x009922**
- restructure recipes to be more compliant with #130 
- extend "maintenance" section in the root README
- include `lint` command into CI/CD
- fix lint errors

Signed-off-by: 0x009922 <a.marcius26@gmail.com>

### CODE REVIEW

**Review by stefashkaa — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -45,12 +45,41 @@ Check out [Hyperledger Iroha 2 Tutorial](https://hyperledger.github.io/iroha-2-d
 
 ## Maintenance
 
-### Explore Jake tasks
+Make sure you have installed Node.js v16.17 or v18. As for a package manager, this project uses [PNPM](https://pnpm.io/).
```
Comment (outoftardis) [inline README.md:48]: not v17?

File: README.md [hunk 2]
```diff
@@ -45,12 +45,41 @@ Check out [Hyperledger Iroha 2 Tutorial](https://hyperledger.github.io/iroha-2-d
...
+
+Before working with the repo, install packages:
```
Comment (outoftardis) [inline README.md:50]:
```suggestion
Before working with the repository, install the necessary packages:
```

File: README.md [hunk 3]
```diff
@@ -45,12 +45,41 @@ Check out [Hyperledger Iroha 2 Tutorial](https://hyperledger.github.io/iroha-2-d
...
+# Perform type-checking
+pnpm type-check
+
+# Check/fix lint errors
+pnpm lint
+pnpm lint --fix
+
+# Fix formatting
+pnpm format:fix
+```
```
Comment (outoftardis) [inline README.md:79]: - Run all project tests:

  ```bash
  pnpm test
  ```

- Perform type-checking:

  ```bash
  pnpm type-check
  ```

- Check or fix lint errors:

  ```bash
  pnpm lint
  pnpm lint --fix
  ```

- Fix formatting:

  ```bash
  pnpm format:fix
  ```

File: README.md [hunk 4]
```diff
@@ -45,12 +45,41 @@ Check out [Hyperledger Iroha 2 Tutorial](https://hyperledger.github.io/iroha-2-d
...
+
+Each monorepo package might have its own scripts and tasks. Please refer to a specific package's README for details.
```
Comment (outoftardis) [inline README.md:81]:
```suggestion
Each monorepo package might have its own scripts and tasks. You can find the package details in a README for that package.
```

File: README.md (hunk 1 above)
Comment (0x009922) [reply README.md:48]: Odd versions of Node.js are unstable, even - long-term supported.

In general, it is not very important to mention a lot of versions here - v16.16 is enough.

Comment (outoftardis) [reply README.md:48]: ok

**Review by safinsaf — APPROVED** [review]

**Pull request merged**